.changeset/add_maxhostcollateral_helper.md

```diff
@@ -0,0 +1,5 @@
+---
+default: patch
```

Comment:
Add another changeset with a breaking change for removing the duration parameter.